<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background : #3BB143;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>711</width>
      <height>481</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background : #50C878;
border-radius : 20px;
color : #000000;</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>6</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>7</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>8</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>9</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>10</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>11</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>12</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>13</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>14</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>15</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>16</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>17</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>18</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>19</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>20</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Kode Pertanyaan</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Penyakit</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>R0001</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>T0001,T0002,T0003,T0004,T0014,T0027,T0028,T0034</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>Ulkus Kornea</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>R0002</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>T0001,T0014,T0018,T0021</string>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>Keratokonus</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>R0003</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>T0005,T0006,T0007,T0008</string>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string>Kalazion</string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>R0004</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string>T0001,T0005,T0009,T0010,T0011,T0012,T0027,T0028</string>
     </property>
    </item>
    <item row="3" column="2">
     <property name="text">
      <string>Blefaritis</string>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string>R0005</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="4" column="1">
     <property name="text">
      <string>T0013,T0014,T0015, T0027</string>
     </property>
    </item>
    <item row="4" column="2">
     <property name="text">
      <string>Herdeolum (Stye)</string>
     </property>
    </item>
    <item row="5" column="0">
     <property name="text">
      <string>R0006</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="5" column="1">
     <property name="text">
      <string>T0001,T0002,T0012,T0014,T0028</string>
     </property>
    </item>
    <item row="5" column="2">
     <property name="text">
      <string>Konjungtivitas</string>
     </property>
    </item>
    <item row="6" column="0">
     <property name="text">
      <string>R0007</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="6" column="1">
     <property name="text">
      <string>T0005,T0016,T0017</string>
     </property>
    </item>
    <item row="6" column="2">
     <property name="text">
      <string>Trakoma</string>
     </property>
    </item>
    <item row="7" column="0">
     <property name="text">
      <string>R0008</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="7" column="1">
     <property name="text">
      <string>T0018,T0019,T0020</string>
     </property>
    </item>
    <item row="7" column="2">
     <property name="text">
      <string>Ablasio Retina</string>
     </property>
    </item>
    <item row="8" column="0">
     <property name="text">
      <string>R0009</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="8" column="1">
     <property name="text">
      <string>T0018,T0019</string>
     </property>
    </item>
    <item row="8" column="2">
     <property name="text">
      <string>Retinopati Diabetikum</string>
     </property>
    </item>
    <item row="9" column="0">
     <property name="text">
      <string>R0010</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="9" column="1">
     <property name="text">
      <string>T0002,T0005,T0014,T0027,T0035,T0043,T0044</string>
     </property>
    </item>
    <item row="9" column="2">
     <property name="text">
      <string>Glaukoma </string>
     </property>
    </item>
    <item row="10" column="0">
     <property name="text">
      <string>R0011</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="10" column="1">
     <property name="text">
      <string>T0001,T0002,T0021,T0022</string>
     </property>
    </item>
    <item row="10" column="2">
     <property name="text">
      <string>Katarak</string>
     </property>
    </item>
    <item row="11" column="0">
     <property name="text">
      <string>R0012</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="11" column="1">
     <property name="text">
      <string>T0001,T0018,T0023</string>
     </property>
    </item>
    <item row="11" column="2">
     <property name="text">
      <string>Uveitis</string>
     </property>
    </item>
    <item row="12" column="0">
     <property name="text">
      <string>R0013</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="12" column="1">
     <property name="text">
      <string>T0002,T0005,T0025,T0026</string>
     </property>
    </item>
    <item row="12" column="2">
     <property name="text">
      <string>Selulitis Orbitalis</string>
     </property>
    </item>
    <item row="13" column="0">
     <property name="text">
      <string>R0014</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="13" column="1">
     <property name="text">
      <string>T0024</string>
     </property>
    </item>
    <item row="13" column="2">
     <property name="text">
      <string>Eksoftalmus</string>
     </property>
    </item>
    <item row="14" column="0">
     <property name="text">
      <string>R0015</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="14" column="1">
     <property name="text">
      <string>T0001,T0002,T0014,T0018,T0027,T0028,T0029</string>
     </property>
    </item>
    <item row="14" column="2">
     <property name="text">
      <string>Keratitis Pungtata Superf</string>
     </property>
    </item>
    <item row="15" column="0">
     <property name="text">
      <string>R0016</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="15" column="1">
     <property name="text">
      <string>T0012,T0028,T0030,T0031</string>
     </property>
    </item>
    <item row="15" column="2">
     <property name="text">
      <string>Alergi Mata Merah
</string>
     </property>
    </item>
    <item row="16" column="0">
     <property name="text">
      <string>R0017</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="16" column="1">
     <property name="text">
      <string>T0001,T0002,T0023,T0034</string>
     </property>
    </item>
    <item row="16" column="2">
     <property name="text">
      <string>Endoftalmitis</string>
     </property>
    </item>
    <item row="17" column="0">
     <property name="text">
      <string>R0018</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="17" column="1">
     <property name="text">
      <string>T0025,T0035,T0036,T0037</string>
     </property>
    </item>
    <item row="17" column="2">
     <property name="text">
      <string>Trombosis Sinus Kavernosu</string>
     </property>
    </item>
    <item row="18" column="0">
     <property name="text">
      <string>R0019</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="18" column="1">
     <property name="text">
      <string>T0038,T0039</string>
     </property>
    </item>
    <item row="18" column="2">
     <property name="text">
      <string>Optic Neuritis</string>
     </property>
    </item>
    <item row="19" column="0">
     <property name="text">
      <string>R0020</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="19" column="1">
     <property name="text">
      <string>T0014,T0025,T0027,T0040,T0041</string>
     </property>
    </item>
    <item row="19" column="2">
     <property name="text">
      <string>Dakriotitis</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Palatino Linotype</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color : #fff
</string>
    </property>
    <property name="text">
     <string>Tabel Rule Penyakit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>889</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
